### ISSUE TITLE
[BUG/Help] <title>

### ISSUE THREAD

**Issue by fxb392** [THUDM/ChatGLM2-6B#241]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

q

### Steps To Reproduce

q

### Environment

```markdown
- OS:
- Python:
- Transformers:q
- PyTorch:
- CUDA Support (`python -c "import torch; print(torch.cuda.is_available())"`) :
```

**Comment by lilongxian** [comment]
tokenizer = ChatGLMTokenizer.from_pretrained(model_dir)
print(tokenizer.special_tokens)  # {'<bos>': 1, '<eos>': 2, '<pad>': 0}
print(tokenizer.tokenizer.special_tokens)   # {'[MASK]': 64789, '[gMASK]': 64790, '[sMASK]': 64791, 'sop': 64792, 'eop': 64793}

代码：
class ChatGLMTokenizer(PreTrainedTokenizer):
       。。。。。。
        self.tokenizer = SPTokenizer(vocab_file)
        self.special_tokens = {
            "<bos>": self.tokenizer.bos_id,  # 1
            "<eos>": self.tokenizer.eos_id,  # 2
            "<pad>": self.tokenizer.pad_id   # 0
        }
class SPTokenizer:
    def __init__(self, model_path: str):
       。。。。。。
        special_tokens = ["[MASK]", "[gMASK]", "[sMASK]", "sop", "eop"]
        self.special_tokens = {}  # {'[MASK]': 64789, '[gMASK]': 64790, '[sMASK]': 64791, 'sop': 64792, 'eop': 64793}
        self.index_special_tokens = {}
        for token in special_tokens:
            self.special_tokens[token] = self.n_words
            self.index_special_tokens[self.n_words] = token
            self.n_words += 1

**Comment by fxb392** [comment]
但是获取的词表里面为什么没加进去这几个特殊字符呢{'[MASK]': 64789, '[gMASK]': 64790, '[sMASK]': 64791, 'sop': 64792, 'eop': 64793}

> Folder 64780
export 64781
parator 64782
prec 64783
LASS 64784
Attributes 64785
fx 64786
▁fmt 64787
▁attr 64788
